domains/arif.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Personal Portfolio Website",
+  "repo": "https://github.com/Arif-Islam/Portfolio-site",
+  "owner": {
+    "username": "Arif-Islam",
+    "email": "arif8491@gmail.com"
+  },
+  "record": {
+    "CNAME": "Arif-Islam.github.io"
```

`CNAME` record must be lowercased:
```suggestion
    "CNAME": "arif-islam.github.io"
```